.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: php
 
 php:
+  - 7.0
```

Comment:
PHP 7 is up to twice as fast as PHP 5.x 
We can got the tips first time if the unit test is't passed.